beater/server.go

```diff
@@ -62,12 +62,18 @@ func newServer(config Config, publish successCallback) *http.Server {
 	}
 }
 
-func start(server *http.Server, ssl *SSLConfig) {
-	if ssl.isEnabled() {
-		go server.ListenAndServeTLS(ssl.Cert, ssl.PrivateKey)
-	} else {
-		go server.ListenAndServe()
-	}
+func start(server *http.Server, ssl *SSLConfig) chan(error) {
+	c := make(chan(error), 1)
```

Comment:
I prefer this way instead. This is easy to test. `beater.go` is a libbit dependency and doesn't lend itself to much testing.